apps/src/code-studio/pd/workshop_survey/Implementation.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {FormGroup} from 'react-bootstrap';
+import FormComponent from '../form_components/FormComponent';
+
+const LABELS = {
+  "hoursPerWeek": "Roughly how many hours per week do you get with each section of students to teach CS Discoveries?",
+  "weeksPerYear": "How many weeks will you be spending on CS Discoveries with each section of students this year?",
+  "courseStructure": "Which of the following best describes how you are teaching CS Discoveries with each section of students?",
+  "unitsPlanningToTeach": "Which units in CS Discoveries do you plan to teach to your sections this year?",
+  "sameStudentsMultipleYears": "Will you see the same students for multiple years?",
+  "unitsInLaterYears": "Please indicate which units you will teach in the second or third year:",
+  "combiningCurricula": "Are you combining CS Discoveries with other curricula you teach in the same class? If so, what else are you teaching? (check all that apply)",
+  "cteCredit": "Can students get CTE credit for taking CS Discoveries in your school?",
+  "csdRequired": "At your school, is your CS Discoveries course required or optional?"
+};
+
+export default class Implementation extends FormComponent {
+  render() {
+    return (
+      <FormGroup>
+        <p>Now that school has started, please answer these questions so we can understand how you will be using CS Discoveries in your class this year. If you have multiple sections that each use it differently, answer the questions below for the section that will have the longest implementation of CS Discoveries.</p>
```

Comment:
nit: we aren't strongly enforcing line lengths in these components, but it would still be a lot more readable if this was split up:

```
<p>
  Now that school has started, please answer these questions so we can understand
  how you will be using CS Discoveries in your class this year. If you have
  multiple sections that each use it differently, answer the questions below for
  the section that will have the longest implementation of CS Discoveries.
</p>
```